.github/workflows/pr-labels.yml

```diff
@@ -6,7 +6,7 @@ jobs:
   setup-label:
     runs-on: ubuntu-latest
     env:
-      NX_CLOUD_ACCESS_TOKEN: ${{ secrets.NX_CLOUD_ACCESS_TOKEN }}
+      NX_CLOUD_ACCESS_TOKEN: ${{ env.NX_CLOUD_ACCESS_TOKEN }}
```

Comment:
This env does exist in this pipeline? As it is not triggered by the main pipeline